configs/components/augeas.rb

```diff
@@ -90,7 +79,6 @@
       pkg.environment "PKG_CONFIG_PATH", "/opt/csw/lib/pkgconfig"
```

Comment:
I think ideally we'd want to move all these environment variables up to platform files too, but that might be too much for this PR :)

Reply:
There's also no way to force environment from platform files, we would need to add that to vanagon

Reply:
Yeah, we need to enable that - a ton of the duplication across components is environment variables